bats/README.md

```diff
@@ -0,0 +1,143 @@
+## Overview
+
+BATS is a testing framework for Bash shell scripts that provides supporting libraries and helpers for customizable server-side test automation.
+
+## Setup
+
+It's important to have a Rancher Desktop CI build installed with privileged rights before running the BATS tests.
+
+On Windows:
+  Prior to running `git submodule update --init`, it's important to setup the Git configuration by running the following commands:
+
+  ```
+  git config --global core.eol lf
+  git config --global core.autocrlf false
+  ```
+  Then, checkout the Git repository or clone it.
+
+All platforms:
+From the root directory of the Git repository, run the following commands to install BATS and its helper libraries into the BATS test directory:
+
+  ```
+  git submodule update --init
+  ```
+
+## Running BATS
+
+1. To run the BATS test, specify the path to BATS executable from bats-core and run the following commands:
```

Comment:
Sine you talk above about cloning the repo on Windows, you should also explain how to locate it inside a WSL distro on `/mnt/...`.

I see that happens in the 3rd example, but it does not explain that `/mnt/c` represents the `C:` drive from windows.